### PR TITLE
Albumphotos list

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

Albumphotos list

## Trello Card

https://trello.com/c/h5ZqEhhT

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -14,6 +21,7 @@ exports.init = app => {
   app.get('/albums/:id/photos', getAlbumsPhotos);
   app.get('/users', validateTokenMiddleware, userList);
   app.get('/users/:user_id/albums', [validateTokenMiddleware], userAlbumsList);
+  app.get('/users/albums/:id/photos', [validateTokenMiddleware], userAlbumPhotosList);
```
Comment (jersonlopez) [inline app/routes.js]: remove []

File: app/services/users.js [hunk 2]
```diff
@@ -103,3 +105,18 @@ exports.userAlbumsList = async req => {
     throw err;
   }
 };
+
+exports.userAlbumPhotosList = async req => {
+  try {
+    const albumId = req.params.id;
+    const user = await exports.getUser(req.body.decode.email);
+    const purchasedAlbum = await servicesAlbums.albumPurchased(albumId, user.id);
+    if (purchasedAlbum !== true) {
```
Comment (jersonlopez) [inline app/services/users.js]: if(!purchasedAlbum){...}

File: test/albums.test.js [hunk 3]
```diff
@@ -12,6 +12,14 @@ const responseAlbumsList = [
   }
 ];
 
+const albumPhotos = {
```
Comment (jersonlopez) [inline test/albums.test.js]: this object is already define in resquest-promise.js file

File: test/albums.test.js [hunk 4]
```diff
@@ -120,3 +128,92 @@ describe('user Albums List', () => {
...
+              .then(result => {
+                expect(result.statusCode).toBe(200);
+                expect(result.body).toEqual(albumPhotos);
+                done();
+              });
+          })
+      );
+  });
+
+  it('should not list the photos of the albums of a user', done => {
```
Comment (jersonlopez) [inline test/albums.test.js]: the test is good but the description could be better, something like 'should not list the photos of the albums of another user'

**Review by jersonlopez — CHANGES_REQUESTED** [review]

File: app/routes.js (hunk 1 above)
Comment (YesicaNavaB) [reply app/routes.js]: ok!

File: test/albums.test.js (hunk 3 above)
Comment (YesicaNavaB) [reply test/albums.test.js]: moved to  utils.test.js

File: test/albums.test.js (hunk 4 above)
Comment (YesicaNavaB) [reply test/albums.test.js]: perfect!

**Review by jersonlopez — APPROVED** [review]
finally 😃😃😃😃😃 good job!

**Pull request merged**